proxy/utils/headers.ts

```diff
@@ -70,26 +70,26 @@ export function updateResponseHeaders(
 }
 
 function resolveClientIp(request: HttpRequest, logger?: Logger) {
-  const forwardedFor = request.headers['x-forwarded-for']
+  const clientIp =
+    request.headers['x-azure-clientip'] || request.headers['x-client-ip'] || request.headers['x-real-ip'] || ''
 
-  if (forwardedFor) {
-    const [clientIp] = forwardedFor.split(',')
+  logger?.verbose('Client IP resolved', {
+    clientIp,
+  })
 
-    logger?.verbose('Client IP resolved from x-forwarded-for', {
-      clientIp,
-      forwardedFor,
-    })
+  return stripPort(clientIp)
+}
 
-    return clientIp
+function stripPort(ip: string) {
+  if (!ip.includes(':')) {
```

Comment:
Won't work as expected for IPv6 addresses.

Reply:
Thanks, good catch! Fixed 🙂